pages/apim/3.x/installation-guide/docker/installation-guide-docker-compose-quickstart.adoc

```diff
@@ -0,0 +1,43 @@
+= Quick Install with Docker Compose
+:page-sidebar: apim_3_x_sidebar
+:page-permalink: apim/3.x/apim_installation_guide_docker_compose_quickstart.html
+:page-folder: apim/installation-guide/docker
+:page-layout: apim3x
+
+This page describes how to install and run APIM in Docker containers on `localhost` using the `docker compose` command.
+
+== Prerequisites
+
+Docker must be installed and running. For more information about installing Docker, see the link:{{'https://www.docker.com/' | relative_url }}[Docker website].
+
+== Installing APIM
+
+1. Download the `docker-compose.yml` file as `docker-compose-apim.yml`.
++
+[code,bash]
+----
+curl https://raw.githubusercontent.com/gravitee-io/gravitee-docker/master/apim/3.x/docker-compose.yml -o docker-compose-apim.yml  
```

Comment:
What's the thinking behind renaming the docker-compose file? With the default name, you can run commands like `docker compose up -d` without having to include the filename

Reply:
It means the file is less likely to be overwritten if someone tries to download another `docker-compose.yml` file.

For example, if someone was setting up Kafka using `docker compose` they wouldn't risk overwiritng `docker-compose-apim.yml`. While this is unlikely to happen in this case, it did happen when someone was reading the event-native tutorials, so I think it is important to keep here, if only for consistency's sake.